FrontEnd/FrontEnd.Web/Components/Pages/Recipes/SingleRecipe.razor

```diff
@@ -101,6 +101,18 @@ else
             </ol>
         </FluentGridItem>
     </FluentGrid>
+    @if (_relatedRecipes != null)
+    {
+        <FluentLabel Typo="Typography.H3" MarginBlock="30px 10px">Related recipes</FluentLabel>
```

Comment:
Perhaps matching label to accent color would look more consistent with the overall page labels.